spectator/__init__.py

```diff
@@ -1,6 +0,0 @@
-from .clock import Clock        # noqa: F401
-from .clock import ManualClock  # noqa: F401
-from .clock import SystemClock  # noqa: F401
-from .registry import Registry
-
-GlobalRegistry = Registry()
```

Comment:
Wouldn't this break uses like:

```python
from spectator import GlobalRegistry
```

Reply:
Absolutely. These changes are going to be released as 1.0.0, compared to the current 0.2.10.

I think it is important to get rid of the last vestigial bits of the thick client and get this library to be more in-line with the other thin clients. I rather liked how the spectator-go thin client turned out, and a number of ideas from that helped make the API here much cleaner.

A migration guide will be provided and we will start with producing a release candidate build. We can then use that to attempt a migration for a few sample apps, to ensure that we have an accurate migration guide.

Reply:
There is [a lot of usage](https://sourcegraph.netflix.net/search?q=context%3Aglobal+GlobalRegistry&patternType=keyword&sm=0&filters=%5B%5B%22lang%22%2C%22Python%22%2C%22lang%3Apython%22%5D%5D), I think this would make adoption significantly slower.

Reply:
We can keep the `GlobalRegistry` concept to help ease adoption.